lib/rubygems/ext/cargo_builder.rb

```diff
@@ -4,26 +4,26 @@
 # over the `cargo rustc` command which takes care of building Rust code in a way
 # that Ruby can use.
 class Gem::Ext::CargoBuilder < Gem::Ext::Builder
-  attr_reader :spec
+  attr_reader :spec, :runner
 
-  def initialize(spec)
+  def initialize(spec, runner: nil)
     @spec = spec
+    @runner = runner || self.class.method(:run)
   end
 
   def build(_extension, dest_path, results, args = [], lib_dir = nil, cargo_dir = Dir.pwd)
     require "rubygems/command"
+    require "rubygems/ext/cargo_builder/link_flag_converter"
```

Comment:
Can we use relative requires for internal files, to be consistent with the rest of the code?

Reply:
Thanks! Could you also change the previous `require`? It was not introduced here but in #5175, but I happened to notice it just now.